themes/default/content/pulumi-up/_index.md

```diff
@@ -94,100 +73,12 @@ schedule_overview:
-
-  - title: Deploying Microservices With Pulumi & AWS Lambda
-    url: https://webinars.devops.com/pulumi-aws-workshop?utm_campaign=%242022.06.28%24_Pulumi_Workshop_DO&utm_source=Pulumi
-    description: |
-      In this workshop, we’ll examine how Pulumi can rapidly accelerate provisioning of cloud infrastructure. We’ll focus on AWS Lambda and build an example set of microservices utilizing AWS’s newest Lambda features.
-
-  - title: Getting Started with Infrastructure as Code on DigitalOcean
-    url: /resources/getting-started-with-infrastructure-as-code-on-digital-ocean/
-    description: |
-      In this hands-on workshop, you will learn how to stand up basic services using Infrastructure as Code through a series of hands-on labs.
-
 registration:
-    sponsor:
-        - hubspot_form_id: ""
-          description: |
-            I'd like to be entered to receive swag from PulumiUP sponsors.
-    main:
-        - hubspot_form_id: "92ba7cbd-6d2e-4845-b5ae-ab964a8d01ad"
-          title: "PulumiUP"
-          description: |
-            PulumiUP is for anyone who is interested in cloud engineering, cloud infrastructure,
-            software development, modern applications, or Pulumi. Whether you're a seasoned cloud
-            engineer or just curious to learn what cloud engineering is all about, PulumiUP will
-            teach you something new about the future and fundamentals of building on the cloud.
-
-        - hubspot_form_id: "b4720fc1-c445-4ffd-a8b6-c167370ce398"
-          title: "Community Summit"
-          description: |
-            Join your friends in the Pulumi community for the first ever Pulumi Community Summit.
-            This virtual event will include an "Open Spaces" concept where you can propose and
-            vote on topics ahead of time. Those topics with the most interest will form the agenda
-            for the day, with virtual breakout rooms centered around those themes.
-
-        - hubspot_form_id: "1103e1e2-8a6d-4664-a006-14b27265adfd"
-          title: "Workshop Series"
-          description: |
-            Following PulumiUP and The Community Summit, Pulumi will be hosting a series of workshops
-            geared at helping you improve your cloud engineering skills. We will have a variety workshops
-            covering basic and advanced concepts.
-
-        - hubspot_form_id: "ce4963db-8014-4282-8418-e5dd6639d21b"
-          title: Want swag from sponsors?
-          description: |
-            I'd like to be contacted by sponsors about swag and other offers.
+    items:
+      - hubspot_form_id: "6ca8d6a5-2824-4ee4-a201-26369267dc11"
+        title: "PulumiUP"
+        description: |
+          none
 
 customer_logos:
   title: Brought to you by
```

Comment:
```suggestion
  title: 2022 was brought to you by
```

I think we should explicitly label the sponsors were for 2022, to avoid confusion with current sponsorship.

Reply:
Oh yeah, agree.